src/Package/Test/DataInspect/EvaluationWrapperTest.cs

```diff
@@ -87,7 +96,7 @@ public class EvaluationWrapperTest : IAsyncLifetime {
                 CanShowDetail = expectedCanShowDetail
             };
 
-            using (var hostScript = new VariableRHostScript()) {
+            using (var hostScript = new VariableRHostScript(_sessionProvider, _broker.BrokerConnector)) {
```

Comment:
Looks like we need `VariableRHostScript` in every test, so we can create it in `InitializeAsync` and dispose in `DisposeAsync`